packages/client/lib/net/server/rlpxserver.ts

```diff
@@ -122,17 +122,17 @@ export class RlpxServer extends Server {
   async bootstrap(): Promise<void> {
     const promises = this.bootnodes.map((node) => {
       const bootnode = {
-        address: node.ip,
+        address: node.ip!,
         udpPort: node.port,
         tcpPort: node.port,
       }
-      try {
-        return this.dpt!.bootstrap(bootnode)
-      } catch (e) {
-        this.error(e)
-      }
+      return this.dpt!.bootstrap(bootnode)
     })
-    await Promise.all(promises)
+    try {
+      await Promise.all(promises)
+    } catch (e) {
+      this.error(e)
```

Comment:
codecov is warning that this line isn't covered, it's not *so* important, but could add a quick test forcing a fast timeout to ensure the error is caught and propagated properly

Reply:
I will leave this for now but generally that's a good idea - also left a comment on the codecov/patch discussion in the chat. Will likely be good if we get to some work mode where we take coverage more serious again (I at least started in #1028 to add some substantial tests and had a look at codecov/patch, took me quite some time but I think I finally had some mental breakthrough in my `testdouble` understanding 😄  ).

Reply:
I've directly taken the occasion and made the `codecov/project` check mandatory again, together with the other library's `test-*` checks (e.g. `test-trie`), think we still had some gap here due to uncertainties in the CI setup for some time (these "run everything or run selected" discussions). And we can of course later always adopt again if some CI setup change comes up.